tic_tac_toe/game.py

```diff
@@ -1,104 +1,131 @@
+from .exceptions import *
+
 # internal helpers
 def _position_is_empty_in_board(position, board):
-    """
-    Checks if given position is empty ("-") in the board.
-
-    :param position: Two-elements tuple representing a
-                     position in the board. Example: (0, 1)
-    :param board: Game board.
-
-    Returns True if given position is empty, False otherwise.
-    """
-    pass
+    x, y = position    
+    return board[x][y] == '-'
 
 
 def _position_is_valid(position):
-    """
-    Checks if given position is a valid. To consider a position as valid, it
-    must be a two-elements tuple, containing values from 0 to 2.
-    Examples of valid positions: (0,0), (1,0)
-    Examples of invalid positions: (0,0,1), (9,8), False
-
-    :param position: Two-elements tuple representing a
-                     position in the board. Example: (0, 1)
-
-    Returns True if given position is valid, False otherwise.
-    """
-    pass
+    if not isinstance(position, tuple):
+        return False
+    
+    valid_pos = [(x,y) for x in range(3) for y in range(3)]
+    
+    return position in valid_pos
 
 
 def _board_is_full(board):
-    """
-    Returns True if all positions in given board are occupied.
-
-    :param board: Game board.
-    """
-    pass
+    return not any('-' in row for row in board)
```

Comment:
We like the clean list comprehension one liner for board_is_full function. Well done.